src/extensions/clinvar/tables/ClinicalAssertionVariation.json

```diff
@@ -27,6 +23,17 @@
         }
       ]
     },
+    {
+      "name": "parent_ids",
+      "datatype": "integer",
```

Comment:
Is this an ID that's already present in the XML, or is this the one they said we should generate? If it's generated, could you ask if there's an alternate field / combo of fields that'd make sense to use as the natural PK?

Generating IDs is going to be tough with Jade's implementation of upsert, since it relies on a constant primary key for each row.

Reply:
these are the linkage/lineage IDs of all a given row's parents. They are generated from the XML